activerecord-sqlserver-adapter.gemspec

```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
   spec.add_dependency 'activerecord', '~> 5.0.0'
-  spec.add_dependency 'tiny_tds'
+  spec.add_dependency 'ruby-odbc'
```

Comment:
Not sure this is a good idea, won't it cause some of the code in this adapter not to work? I'm sure the adapter references TinyTDS classes. Can we fix the gem instead of removing this? Don't we also want to work toward supporting both modes? We'd have to undo this later yeah?

Reply:
Here we are just updating the adapter's dependency to use ruby-odbc instead of tiny_tds and NOT removing the tiny_tds gem itself. In this case:
1. The tinytds gem will be installed as the adapter's Gemfile still has tinytds gem
2. The forked activerecord-sqlserver-adapter still supports both modes, the primary being ODBC
3. All test cases pass, in this forked adapter, in dblib mode as well which means the adapter's reference to TinyTDS classes all work fine.
4. In 2.17, we are using only ODBC mode and we can go with this temporary fix for now 
5. In AVM, when using this forked adapter, we will still be able to support both modes by specifying both the tiny_tds and ruby-odbc gems in AVM's Gemfile

I agree fixing the tinytds gem is the right approach and Sukeerthi was working on it earlier. We can use this temporary solution until we get this fixed in the tinytds gem